starter-code/scripts/models/article.js

```diff
@@ -87,10 +92,10 @@
     webDB.execute(
       [
         {
-          // TODO: Delete an article instance from the database based on its id:
+          // DONE: Delete an article instance from the database based on its id:
           /* Note: this is an advanced admin option, so you will need to test
               out an individual query in the console */
-          'sql': '', // <--- complete the command here, inside the quotes;
+          'sql': 'DELETE FROM articleTable WHERE id = this.id;', // <--- complete the command here, inside the quotes;
```

Comment:
'this.id' is not recognized by the SQL database. You need to use '?' to indicate the placement of a variable.